sandbox/settings.py

```diff
@@ -578,6 +578,9 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         environ_prefix=None,
     )
 
+    # Feature flags
+    FEATURES = values.DictValue(environ_name="FEATURES", environ_prefix=None)
```

Comment:
You could add default values into `dev.dist` and `dev-ssl.dist` env files. In this way by default.